CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 Version 0.17.0 *(In development)*
 ---------------------------------
+- Move `excludes` values to the `JunitJacocoExtension` configuration stage, to allow easier modification.
```

Comment:
Can you revert the changes here? I'm generating this file when doing a new release.

Reply:
Done! Also merged my commits to be cleaner.